ci/prow/make_config.go

```diff
@@ -1355,8 +1355,10 @@ func generateTestGroup(repoName string, jobNames []string) {
 			extras["short_text_metric"] = "coverage"
 			// Do not alert on coverage failures (i.e., coverage below threshold)
 			extras["num_failures_to_alert"] = "9999"
+		case "istio-1.0.7-mesh", "istio-1.0.7-no-mesh", "istio-1.1.2-mesh", "istio-1.1.2-no-mesh":
+			extras["alert_stale_results_hours"] = "3"
```

Comment:
Should we also alert after 3 failures, not on every failure, like the default continuous flow? If so, add `extras["num_failures_to_alert"] = "3"` here.

Reply:
done.